### PR TITLE
src/stores: add Quasar integration for Pinia store

### PR DESCRIPTION
**Pull request by maceksimon**
Install `pinia` and `pinia-plugin-persistedstate`.
Create integration for Pinia in `src/stores/index.ts`.
Add test boilerplate for login store and use it in the `FormLogin` component.

### CODE REVIEW

File: src/stores/login.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { defineStore } from 'pinia';
+
+export const useLoginStore = defineStore('login', {
+  state: () => ({
+    user: {
+      email: 'test@example.com',
```
Comment (tmszi) [inline src/stores/login.ts:6]: Email key should has `null` value by default.

For testing purpose set user email key value before running test.

Comment (maceksimon) [reply src/stores/login.ts]: Updated. Linked the login form with store state and added tests that check the store state after typing in values.

File: src/stores/login.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { defineStore } from 'pinia';
+
+export const useLoginStore = defineStore('login', {
+  state: () => ({
+    user: {
+      email: '',
```
Comment (tmszi) [inline src/stores/login.ts:6]: Default value should be `null`, no (same for password key)?

Comment (maceksimon) [reply src/stores/login.ts]: Ok, updated.
But the value needs to be passed into an input as a `string`, so I changed the getter to the "string" getter and sanitized the type there.

**Pull request merged**